constants/constants.py

```diff
@@ -21,27 +20,12 @@
 
 COMPLEX_TYPES = {RECORD, ENUM, ARRAY, MAP, FIXED}
 
-
-MCS_PER_SECOND = 1000000
-MCS_PER_MINUTE = MCS_PER_SECOND * 60
-MCS_PER_HOUR = MCS_PER_MINUTE * 60
-
-MLS_PER_SECOND = 1000
-MLS_PER_MINUTE = MLS_PER_SECOND * 60
-MLS_PER_HOUR = MLS_PER_MINUTE * 60
-
-DAYS_SHIFT = datetime.date(1970, 1, 1).toordinal()
-
+PROJECT_ROOT = os.path.join(os.path.dirname(__file__), '..')
 
 
-def iterate_over_schemata(schema_root):
-	for namespace in [f for f in os.listdir(schema_root) if not f.startswith('.')]:
-		for schema_name in [f for f in os.listdir(os.path.join(schema_root, namespace)) if not f.startswith('.')]:
-			for version in [
-				f for f in os.listdir(os.path.join(schema_root, namespace, schema_name)) if not f.startswith('.')
-			]:
-				yield schema_name, int(version)
+SCHEMA_ROOTS = [os.path.join(PROJECT_ROOT, 'cerializer', 'tests', 'schemata')]
 
 
-MODE_SERIALIZE = 'serialize'
-MODE_DESERIALIZE = 'deserialize'
\ No newline at end of file
+class SerializationMode(enum.Enum):
+	MODE_SERIALIZE = 'serialize'
+	MODE_DESERIALIZE = 'deserialize'
```

Comment:
Do you have now more constants than this when you deleted `data_constants`? Because it would be nicer to use just `constants.BOOLEAN` than `constants.constants.BOOLEAN`